.github/workflows/ci.yml

```diff
@@ -20,6 +20,10 @@ jobs:
         uses: actions/setup-python@v4
         with:
           python-version: '3.8'
+      - name: Install Python 3.7
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.7'
```

Comment:
The cookiecutters repo's own `ci.yml` needs to install all versions of Python listed in `pyramid-app/cookiecutter.json` and `pypackage/cookiecutter.json`. We just have to manage this manually, I don't think there's any way to automate this that would be worth the cost.